api/src/Entity/DayResponsible.php

```diff
@@ -31,6 +32,16 @@
         new GetCollection(
             security: 'is_authenticated()'
         ),
```

Comment:
As far as I see, you added the new GetCollections, but also kept the old, unfiltered ones everywhere. Shouldn't we switch to the new endpoints completely where possible? I think performance-wise we also found out that it doesn't make sense to fetch e.g. all schedule entries across all accessible camps, so this would enable us to disallow fetching the unfiltered collection, no?

Reply:
I'd agree, that eventually, we should remove the unfiltered GetCollection (if everyone else agrees as well).

However, some additional works is necessary.

`DayResponsibles` is also linked from
- CampCollaboration
- Period

`ScheduleEntry` is also linked from
- Activity
- Day

`Day` is not linked from any other entity in the backend, but in the frontend we [use the `/days` endpoint](https://github.com/ecamp/ecamp3/blob/devel/frontend/src/views/camp/Dashboard.vue#L251) to load all days of a camp.

Overall, I'd vote for keeping this PR limited to introducing subresources as a concept. And further down the road, as we introduce more subresources we can (carefully) start to remove the main GetCollection endpoints.